src/modes/free_for_all.cpp

```diff
@@ -186,9 +188,12 @@ bool FreeForAll::isRaceOver()
 
     if (!getKartAtPosition(1))
         return false;
-    int top_id = getKartAtPosition(1)->getWorldKartId();
+
+    const int top_id = getKartAtPosition(1)->getWorldKartId();
+    const int hit_capture_limit = race_manager->getHitCaptureLimit();
+
     return (m_count_down_reached_zero && race_manager->hasTimeTarget()) ||
-        m_scores[top_id] >= race_manager->getHitCaptureLimit();
+        (hit_capture_limit != 0 && m_scores[top_id] >= hit_capture_limit);
```

Comment:
I wonder in which case hit_capture_limit can be 0 ?

Reply:
Now you can set the hit capture limit to zero to only use a time limit, like it is with soccer for the goals.
For offline play I set the hit capture limit to zero